### ISSUE TITLE
Method is not a member of Moya 

### ISSUE THREAD

**Issue by mrfarukturgut** [Moya/Moya#436]
There is this error at this part of the extension that is in example.mp 

```
var method: Moya.Method {
        // all requests in this example will use GET.  Usually you would switch
        // on the enum, like we did in `var path: String`
        return .GET
    }
```
`'Method' is not a member of Moya `

Any help? 

And seperately, Can I use Moya with Heimdallr? As I understand Moya can make requests and using Alamofire for this. But I found Heimdallr and the author claims that Heimdallr can refresh tokens by itself. I have issues with task in Alamofire. This is whay I am asking this.

**Comment by mrfarukturgut** [comment]
Solved the main question. A simple mistake. But second question is still valid

**Comment by sunshinejr** [comment]
Hey! Thank you for reporting an issue!. 🎉

With your problem could you please provide what was the case and how you have repaired it? It might be useful for other people in the future ✌️

And about Heimdller, I didn't really play with that, but what I can tell you is that Moya relies on Alamofire in its core. If you want to use Heimdller instead of Alamofire, you would need to create your own version of Moya and replace the Alamofire code in requests etc. But you can try using `endpointClosure` to add additional parameters based on Heimdller. There is an example [here](https://github.com/Moya/Moya/blob/master/docs/Endpoints.md#from-target-to-endpoint-), that adds auth token to your request. Hope it helps! 😉

**Comment by mrfarukturgut** [comment]
Thank you for the information. Alamofire is brilliant framework I know that. But like I said, I am not able to handle 401 http code and I don't know iOS development to solve it. The reason why I want to use Heimdallr is the author of it claims that the Heimdallr will handle it by itself. 
However I will consider your suggestion. Time will show me how I am gonna do :)

And about the first question. Before using `Moya` it turned out I had created a class and called it 'Moya'. When the code tried to reach `Moya.Method`, it threw an error because the class which I called 'Moya' had not a member called `Method`. So It wasn't about the actual `Moya`. :)
